src/caret_analyze/value_objects/callback.py

```diff
@@ -356,6 +460,32 @@ def __init__(
         *,  # for yaml reader only.
         callback_name: str | None = None,
     ) -> None:
+        """
+        Construct an instance.
+
+        Parameters
+        ----------
+        callback_id : str
+            Callback unique id,
+            a value that can be identified when retrieved from the Architecture reader.
+        node_name : str
+            Node name.
+        node_id : str
+            Node unique id,
+            a value that can be identified when retrieved from the Architecture reader.
+        symbol : str
+            Symbol name of the service callback.
+        service_name : str
+            Service name which the service callback service.
+        publish_topic_names : tuple[str, ...] | None
+            Topic name which the service callback publishes.
+        construction_order : int
+            Order of instance creation within the identical node.
+        callback_name : str | None
+            Service callback name, by default None.
```

Comment:
It is better that "Service callback name, by default None." is "Service callback name (: default is None)." 
However, there are many same sentence (, by default None), so if changes are made in this way, everything needs to be changed.

Reply:
This grammar has been used for previous, so there is no need to change it this time, I think.

Reply:
I see. Then I don't think a fix is needed.